.github/workflows/ci.yml

```diff
@@ -0,0 +1,208 @@
+name: CakePHP CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  testsuite:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: ['7.2', '7.4', '8.0']
+        db-type: [sqlite, mysql, pgsql]
+        prefer-lowest: ['']
+        include:
+          - php-version: '7.2'
+            db-type: 'mysql'
+            prefer-lowest: 'prefer-lowest'
+
+    steps:
+    - name: Setup MySQL latest
+      if: matrix.db-type == 'mysql' && matrix.php-version != '7.2'
+      run: docker run --rm --name=mysqld -e MYSQL_ROOT_PASSWORD=root -e MYSQL_DATABASE=cakephp -p 3306:3306 -d mysql --default-authentication-plugin=mysql_native_password --disable-log-bin
```

Comment:
Fetching specific myself versions using docker is quite wasteful here. Just use whatever is provided by the distro.

Reply:
We would need to add the db setup back in since the service starts with whatever configuration ga provides.

Reply:
I reduced the configs down to a single db for each.